### PR TITLE
Fixed codelyzer path

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,14 +1,21 @@
 {
   "rulesDirectory": [
-    "node_modules/codelyzer"
+    "codelyzer"
```
Comment (Blackbaud-BobbyEarl) [inline tslint.json]: Hey @Blackbaud-SteveBrush any insight into this change?

File: tslint.json [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
   ],
   "rules": {
     "member-access": true,
     "member-ordering": [
       true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+      {
```
Comment (Blackbaud-BobbyEarl) [inline tslint.json]: This seems consistent with https://palantir.github.io/tslint/rules/member-ordering/, is this an updated syntax to what we currently have?

Comment (Blackbaud-SteveBrush) [inline tslint.json]: Yes.

File: tslint.json (hunk 1 above)
Comment (Blackbaud-SteveBrush) [inline tslint.json]: This change fixes the issue of requiring SPAs to have their own version of codelyzer listed in package.json. "These values are handled using node module resolution semantics..."

Comment (Blackbaud-SteveBrush) [inline tslint.json]: https://palantir.github.io/tslint/usage/configuration/

**Pull request merged**